pmm/pmm2-testsuite.groovy

```diff
@@ -42,6 +42,7 @@ void runTAP(String TYPE, String PRODUCT, String COUNT, String VERSION) {
             export PMM_VERSION=${PMM_VERSION}
 
             sudo chmod 755 /srv/pmm-qa/pmm-tests/pmm-framework.sh
+            sudo yum install -y jq
```

Comment:
I think it makes sense to move it to `aws-staging-start` job on `Run VM` stage. jq is a cool tool and I think it'll be better to install all tools in one place

Reply:
sure makes sense, I fixed your comment please review again